### ISSUE TITLE
NotFoundException: Resource ID #0x0

### ISSUE THREAD

**Issue by oguzkocer** [wordpress-mobile/WordPress-Android#11617]
@khaykov Another 14.6-rc-1 crash, the line this is crashing doesn't seem to be changed recently, but maybe it's related to the material changes?

**Sentry Url**: https://sentry.io/share/issue/ff8cb93e6b0045e882cfc3619a1cab53/
**User Count**: 1
**Count**: 6
**First Release**: org.wordpress.android-14.6-rc-1
**First Seen**: 2020-04-08T10:31:54Z
**Last Release**: org.wordpress.android-14.6-rc-1
**Last Seen**: 2020-04-08T12:23:10Z
**24 Hours**: 6
**30 Days**: 6

**Stacktrace**:
```
NotFoundException: Resource ID #0x0
    at android.content.res.ResourcesImpl.getValueForDensity(ResourcesImpl.java:225)
    at android.content.res.Resources.getDrawableForDensity(Resources.java:887)
    at android.content.res.Resources.getDrawable(Resources.java:827)
    at android.content.Context.getDrawable(Context.java:626)
    at androidx.core.content.ContextCompat.getDrawable(ContextCompat.java:455)
    at org.wordpress.android.util.ColorUtils.setImageResourceWithTint(ColorUtils.kt:24)
    at org.wordpress.android.ui.media.MediaGridAdapter.onBindViewHolder(MediaGridAdapter.java:196)
    at org.wordpress.android.ui.media.MediaGridAdapter.onBindViewHolder(MediaGridAdapter.java:54)
    at androidx.recyclerview.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:7065)
    at androidx.recyclerview.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:7107)
...
(64 additional frame(s) were not displayed)
```

**Comment by khaykov** [comment]
This is not related to material theme, but I think I know what happens - we try to get a placeholder image in Media Browser,[ based on what type of attachment we think it is](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/util/WPMediaUtils.java#L355). If we don't know what type it is, we return 0 instead of resource id, hence `NotFoundException: Resource ID #0x0`.  Maybe we can return some generic image, like question mark when we don't know what attachment it is.

**Comment by oguzkocer** [comment]
@khaykov That makes sense to me. Maybe we can ask a designer for which icon to use. I don't really know who works on what anymore, so if you agree, do you mind pinging a designer for it? (here or on Slack)

**Comment by oguzkocer** [comment]
@khaykov Were you able to follow up on this? If you can't do it, can you please let me know so I can put this into the `Requires Triage` queue?

**Comment by khaykov** [comment]
@oguzkocer Sorry, this slipped through my radar. I followed up on the icon and will submit fix asap.

**Comment by oguzkocer** [comment]
@khaykov You can submit the fix to `release/14.7`.

**Comment by designsimply** [comment]
This looks ok to close since it appears to have been fixed in 14.7. Props @khaykov. 🎉

Let me know if I've closed too soon though!
